analytics/demo/demo.go

```diff
@@ -165,12 +165,16 @@ func GenerateDemoData(days, recordsPerHour int, orgID string, trackPath bool, wr
 			} else {
 				volume = randomInRange(300, 500)
 			}
+			timeDifference := 3600 / volume // this is the difference in seconds between each record
+			nextTimestamp := ts             // this is the timestamp of the next record
 			for i := 0; i < volume; i++ {
 				r := GenerateRandomAnalyticRecord(orgID, trackPath)
 				r.Day = ts.Day()
```

Comment:
@mativm02 shouldn't Day, Month, Year, Hour fields be calculated with `nextTimestamp`?